consul/datadog_checks/consul/consul.py

```diff
@@ -245,13 +263,15 @@ def check(self, instance):
 
         peers = self.get_peers_in_cluster(instance)
         main_tags = []
+        service_check_tags = []
         agent_dc = self._get_agent_datacenter(instance, instance_state)
 
         if agent_dc is not None:
             main_tags.append('consul_datacenter:{0}'.format(agent_dc))
 
         for tag in instance.get('tags', []):
             main_tags.append(tag)
```

Comment:
Just a nit, we could avoid the `for` loop & appending each tag in the list by using `main_tags.extend(tags)` and `service_check_tags.extend(tags)`.